README.md

```diff
@@ -132,11 +146,11 @@ Teapot has a simple logger that will log out certain things under the hood. This
 
 The default log level is `.none` - That is, logs will neither be generated nor printed out. 
 
-Other log levels, in ascending order of how much log barf they fill your console with, are: 
```

Comment:
@designatednerd big fan of scatological analogy?  😅

Reply:
Log barf is a technical term 😇